spec/lucky/log_handler_spec.cr

```diff
@@ -112,6 +118,30 @@ describe Lucky::LogHandler do
       called.should be_true
     end
   end
+
+  context "when configured with custom log formatter" do
+    it "logs emoji" do
+      begin
+        Lucky::LogHandler.configure do
+          settings.log_formatter = EmojiLogFormatter.new
+        end
+
+        called = false
+        log_io = IO::Memory.new
+        context = build_context("PATCH")
+
+        call_log_handler_with(log_io, context) { called = true }
+
+        log_output = log_io.to_s.chomp
+        log_output.should eq "🐵 PATCH - BOOM"
```

Comment:
I think EmojiFormatter should be the default... :P

Reply:
lol